docs/BackupAndRestore.md

```diff
@@ -1,5 +1,39 @@
 # NuoDB Backup and Restore in Kubernetes
 
+<details>
+<summary>Table of Contents</summary>
+<!-- TOC -->
```

Comment:
auto-generated? Nice!

Reply:
I used an extension for VSCode - _huntertran.auto-markdown-toc_